### PR TITLE
Add option to include container names by regex pattern

### PR DESCRIPTION
**Pull request by jeffothy**

### CODE REVIEW

File: app/discovery/events.go [hunk 1]
```diff
@@ -140,6 +142,10 @@ func (e *EventNotif) group(image string) string {
 }
 
 func (e *EventNotif) isAllowed(containerName string) bool {
+	if e.includesPattern != "" {
+		re, _ := regexp.Compile(e.includesPattern)
```
Comment (umputun) [inline app/discovery/events.go:146]: I'd suggest to move it to the constructor and not to compile on each call

Comment (jeffothy) [reply app/discovery/events.go:146]: Fair point, will do.

Comment (jeffothy) [reply app/discovery/events.go]: @umputun Branch updated.

**Comment by umputun** [comment]
@jeffothy can you take a look pls https://github.com/umputun/docker-logger/issues/24 ?

**Comment by jeffothy** [comment]
> @jeffothy can you take a look pls #24 ?

Yes I will take a look

**Pull request merged**